pkgs/by-name/st/stylelint-lsp/package.nix

```
@@ -41,9 +41,9 @@ stdenvNoCC.mkDerivation (finalAttrs: {
 
     mkdir -p $out/{bin,lib/${finalAttrs.pname}}
     mv {dist,node_modules} $out/lib/${finalAttrs.pname}
-    echo "
-      #!${lib.getExe bash}
-      ${lib.getExe nodejs} $out/lib/${finalAttrs.pname}/dist/index.js \$@
+    echo "\
+    #!${lib.getExe bash}
+    ${lib.getExe nodejs} $out/lib/${finalAttrs.pname}/dist/index.js \$@
```

A bit nicer way to do it would be using `wrapProgram`.
For that to work, we also need to mark the source js file as executable.
```suggestion
    chmod a+x $out/lib/${finalAttrs.pname}/dist/index.js
    makeWrapper $out/lib/${finalAttrs.pname}/dist/index.js $out/bin/stylelint-lsp  \
      --prefix "${lib.getExe nodejs}"
```

---

Sorry I screwed up the PR, will close it and fill a new one with a suggested fix.